input/fsh/Bundle.fsh

```diff
@@ -21,14 +21,51 @@ Description: "A collection of data for an mCODE cancer patient."
     tumorMarker 0..* MS and
     cancerRelatedMedicationRequest 0..* MS and
     cancerRelatedProcedure 0..* MS and
-    // Multiple profiles can be used to represent cancer stage. We will combine them into a single element for simplicity.
     cancerStage 0..* MS and
-    // mCODE genomics
     cancerGeneticVariant 0..* MS and
     cancerGenomicsReport 0..* MS and
     geneticSpecimen 0..* MS and
-    genomicRegionStudied 0..* MS
-    // TODO handle comorbidities, waiting on decision for how these are profiled
+    genomicRegionStudied 0..* MS and
+    cancerRelatedComorbidities 0..* MS and
+    vitalSign 0..* MS and
+    coreLaboratory 0..* MS and
+    otherResources 0..1
+
+* entry[cancerPatient] ^short = "Cancer Patient"
+* entry[primaryCancerCondition] ^short = "Primary Cancer Condition"
+* entry[secondaryCancerCondition] ^short = "Secondary Cancer Condition(s)"
+* entry[cancerDiseaseStatus] ^short = "Cancer Disease Status(es)"
+* entry[performanceStatus] ^short = "Performance Status(es) (ECOG or Karnofsky)"
+* entry[tumorMarker] ^short = "Tumor Marker Tests"
+* entry[cancerRelatedMedicationRequest] ^short = "Cancer-RelatedMedicationRequest(s)"
```

Comment:
```suggestion
* entry[cancerRelatedMedicationRequest] ^short = "Cancer-Related MedicationRequest(s)"
```

Reply:
Actually should be Cancer-Related Medication Request(s). Fixed.